From 6c4d206621c53151e4c45d7e87a2eecbb71f9e88 Mon Sep 17 00:00:00 2001
From: Tim S <stahta01@users.sourceforge.net>
Date: Fri, 22 Jan 2016 11:49:25 -0500
Subject: [PATCH 139/159] - ToolsPlus: Changed cmddialog.fbp properties.
 (Thanks stahta01)

Changed FB Project to more closely match the real cpp/h files.
---
 src/plugins/contrib/ToolsPlus/FormBuilder/cmddialog.fbp | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/plugins/contrib/ToolsPlus/FormBuilder/cmddialog.fbp b/src/plugins/contrib/ToolsPlus/FormBuilder/cmddialog.fbp
index bbd30ee..d443556 100644
--- a/src/plugins/contrib/ToolsPlus/FormBuilder/cmddialog.fbp
+++ b/src/plugins/contrib/ToolsPlus/FormBuilder/cmddialog.fbp
@@ -4,6 +4,6 @@
     <object class="Project" expanded="1">
         <property name="bitmaps"></property>
-        <property name="code_generation">C++|XRC</property>
-        <property name="file"></property>
+        <property name="code_generation">C++</property>
+        <property name="file">CmdConfigDialog</property>
         <property name="first_id">1000</property>
         <property name="icons"></property>
@@ -15,5 +15,5 @@
         <property name="use_enum">0</property>
         <property name="use_microsoft_bom">0</property>
-        <object class="Panel" expanded="1">
+        <object class="Dialog" expanded="1">
             <property name="bg"></property>
             <property name="enabled">1</property>
-- 
2.7.1.windows.2

